src/grpc.ts

```diff
@@ -101,13 +101,14 @@ export class GrpcClient {
       this.grpc = options.grpc!;
       this.grpcVersion = '';
     } else {
-      if (semver.gte(process.version, '8.13.0')) {
+      const minimumVersion = '10.0.0'
```

Comment:
can we move it one level up, before `if`? Otherwise the check can be avoided by passing `grpc` in `options`.

Reply:
^^ I mean, before `if ('grpc' in options)` in line 100.

Reply:
good catch, I think the logic is more correct now 👌 I used semver `lt`, and we know if we're greater than `10.0.0`, `grpc-js` support exists.